connect/mirror-client/src/test/java/org/apache/kafka/connect/mirror/MirrorClientTest.java

```diff
@@ -62,6 +65,7 @@ public void testIsHeartbeatTopic() throws InterruptedException, TimeoutException
         assertFalse(client.isHeartbeatTopic("!heartbeats"));
         assertFalse(client.isHeartbeatTopic("source1heartbeats"));
         assertFalse(client.isHeartbeatTopic("source1-heartbeats"));
+        client.close();
```

Comment:
We are calling `close` even though it doesn't do anything just to avoid warnings? Where are the warnings coming from?

Reply:
I had "resource never closed" warnings in Eclipse. Are we interested in fixing those? If no, I'll remove that change.

Reply:
I'd favor focusing on warnings that the build can generate and enforce.

Reply:
Ok, I've removed these changes. Thanks